starfish/intensity_table.py

```diff
@@ -172,6 +172,39 @@ def save(self, filename: str) -> None:
         """
         self.to_netcdf(filename)
 
+    def save_mermaid(self, filename: str) -> pd.DataFrame:
+        """
+        Writes a .csv.gz file in columnar format that is readable by MERMAID visualization
+        software.
+
+        To run MERMAID, follow the installation instructions for that repository and simply
+        replace the data.csv.gz file with the output of this function.
+
+        Parameters
+        ----------
+        filename : str
+            name for compressed-gzipped MERMAID data file. Should end in '.csv.gz'
+
+        See Also
+        --------
+        https://github.com/JEFworks/MERmaid
+
+        """
+
+        # verify the IntensityTable has been decoded
+        if Features.TARGET not in self.coords.keys():
+            raise RuntimeError(
+                'IntensityTable must be decoded before it can be converted to MERMAID input.'
+            )
+
+        # construct the MERMAID dataframe
+        df = self.to_features_dataframe()
+        mermaid_data = df[[Indices.X, Indices.Y, Features.TARGET, Features.TARGET]]
+        mermaid_data.columns = ['x', 'y', 'gene1', 'gene2']
```

Comment:
i just confirmed from the mermaid code/example data, that you can add an arbitrary number of genes here, but they will be the same column. the viz generates a drop down menu and a color for any categorical variable. so this is consistent with the demo currently on the website